bsk_rl/envs/general_satellite_tasking/scenario/sat_actions.py

```diff
@@ -173,3 +173,22 @@ def set_action(self, action: Union[int, Target, str]):
             self.prev_action_key = self.image(action, self.prev_action_key)
         else:
             super().set_action(action)
+
+
+class NadirImagingActions(DiscreteSatAction, ImagingSatellite):
+    def __init__(self, *args, **kwargs) -> None:
+        """
+        Discrete action to image nadir.
+        """
+        super().__init__(*args, **kwargs)
+
+        # Sets a static Nadir target
+        self.targets = Target(name="nadir", location=np.array([0, 0, 0.1]), priority=0)
+        self.add_action(self.nadirImage, n_actions=1, act_name="nadirImage")
+
+    def nadirImage(self, *args, prev_action_key=None, **kwargs) -> None:
+        """Activate Nadir imaging action."""
+
+        if prev_action_key != "nadir_image":
+            self.task_target_for_imaging(self.targets, nadir_pointing=True)
+        return "nadir_image"
```

Comment:
Sorry wasn't very clear here. I think you should delete `self.targets` and the arguments to `self.fsw.action_nadir_scan`, don't see a use case for specifying them since nadir is alway the same place

Reply:
Also `location`, `priority` and `targets` in `NadirTarget(EnvironmentFeatures)` are unneeded. The name `NadirTarget` might make more sense as something else since it's not really representing a target per se